public/app/features/correlations/Forms/TransformationsEditor.tsx

```diff
@@ -0,0 +1,293 @@
+import { css } from '@emotion/css';
```

Comment:
Could you also add some smoke test in integration tests in CorrelationsPage.test.tsx (as steps when adding/editing a correlation)